src/github_utils.py

```diff
@@ -19,111 +20,104 @@ def get_pull_request_diff():
     # Get diff content
     diff_url = pull_request.diff_url
     response = requests.get(diff_url)
-    print(response.text)
+    #print(response.text)
     return response.text
```

Comment:
Remove debugging print statement
```suggestion
|
```
N/A